monitoring.adoc

```diff
@@ -777,17 +777,68 @@ account activity for one user without filling up mailbox.log with log
 messages for unrelated accounts.  See <<command_line_utilities,Command-Line
 Utilities>>, the `zmprov` miscellaneous section.
 
-*Reading records in the log* The example below is a record showing that
-on June 25, 2007, the zimbra server with an IP address of 127.0.0.1 was in
-the process of deleting backups that were created on Monday, June 18, 2007
-at 8 seconds after midnight Pacific Daylight Time (PDT) or older than that
-date.
+*Log pattern*
 
-image:images/mailboxlogdescription.png[Mailbox Log Entry]
+by default log entries in `mailbox.log` have the following Log4j pattern:
+====
+%d %-5p [%t] [%z] %c{1} - %m%n
+====
+This pattern consists of 6 blocks of data:
 
-[NOTE]
-*Component thread number* identifies which thread managed by mailboxd is
-performing the action logged.
+* Date and time (e.g.: `2018-01-22 19:23:07,100`)
+* Log level (e.g. `INFO`)
+* Thread number with a prefix (e.g. `[qtp1043351526-547:https:https://localhost:7071/service/admin/soap/DeleteAccountRequest]`, `[Index-9]`, etc)
+* Zimbra context
+* Component name (e.g. `soap`, `mailbox`, `mbxmgr`, etc)
+* Log message
+
+You can read more about Log4j patterns in https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html[Log4j PatternLayout documentation].
+
+*Thread prefix in mailbox.log*
+
+Thread numbers in mailbox.log are prefixed to identify internal components. The following prefixes are currently used in {product-name}: `btpool`, `pool`, `LmtpServer`, `ImapServer`,`ImapSSLServer`, `Pop3Server`, `Pop3SSLServer`, `ScheduledTask`, `Timer`, `AnonymousIoService`, `CloudRoutingReaderThread`, `GC`, `SocketAcceptor`, `Thread`, `qtp`.
+
+*Zimbra Context in mailbox.log*
+
+`[%z]` section in the log pattern describes Zimbra context and consists of key-value pairs in the format `key=value`, separated by semi-colons (`;`). The following key value pairs are currently supported and may be recorded in log entries in any order and eny combination:
+
+* `ip` - IP of the TCP/IP client making a request
+* `oip` - originating IP address. When a request is made through NGINX proxy, this value will contain the IP address of the client application, while `ip` value will contain the IP address of the proxy server
+* `cid` - connection id of a server that is monotonically increasing - useful for tracking individual connections
+* `id` - ID of the target account
+* `name` - name of the target account (email address)
+* `aid` - ID of the authenticated account. Only present if target account is different then authenticated account
+* `aname` - name of the authenticated account. Only present if target account is different then authenticated account
+* `mid` - Id of requested mailbox. Only present if request is dealing with a mailbox
+* `ua`  - name of the client application (i.e. User Agent)
```

Comment:
looks like we re replacing only ";" with ";;", but not "="

Reply:
probably ok since you can still parse based on the first equal in the block which should be right assuming the name doesn't have a "=" which is currently  the case and controlled by  zimbra so can be guaranteed to be the case.